gitlab/protocols/http_backend_response.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from abc import abstractmethod
```

Comment:
I personally (I think @nejch may have a different opinion) am not a fan of importing methods/classes.

I think just `import abc` is good and then easier when reading the code below to know where it comes from.